src/v3/src/components/HiddenIFrame/HiddenIFrame.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box } from '@mui/material';
+import { FunctionComponent, h } from 'preact';
+
+type HiddenIFrameProps = {
+  id?: string;
+  src: string;
+};
+const HiddenIFrame: FunctionComponent<HiddenIFrameProps> = ({ id, src }) => (
+  <Box
+    id={id}
```

Comment:
Can you just check to make sure this won't render `id="undefined"` when `id` isn't passed?

Reply:
In regards to this, do you mean do a sanity check that the string `"undefined"` doesn't get set as the `id` when the `id` is omitted?

Reply:
@trevoring-okta yeah exactly. Just want to make sure it omits the property instead of the value getting coerced to the string "undefined".

Reply:
Got it, just checked and it omits the `id` 👍